server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java

```diff
@@ -108,6 +112,8 @@ public class IoTDBConfigCheck {
 
   private static final String DATA_NODE_ID = "data_node_id";
 
+  private static final String CONFIG_NODE_LIST = "config_node_list";
+
```

Comment:
Change the class name to `IoTDBStartCheck`?

Reply:
Ok~